core/src/main/java/com/threerings/getdown/data/Application.java

```diff
@@ -547,6 +547,36 @@ public String getTrackingCookieProperty ()
     public Config init (boolean checkPlatform)
         throws IOException
     {
+        Config config = initConfig(checkPlatform);
+        initConfigJava(config);
+        initConfigTracking(config);
+        initConfigResources(config);
+        initConfigArgs(config);
```

Comment:
simply split this long ``init`` to several smaller funcs (have no idea for naming trailing opts so left them here for now)